docs/README-he.md

```diff
@@ -0,0 +1,190 @@
+# ![nuclear](https://i.imgur.com/oT1006i.png) 
+[![Maintainability](https://api.codeclimate.com/v1/badges/a15c4888a63c900f6cc1/maintainability)](https://codeclimate.com/github/nukeop/nuclear/maintainability) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/30750586202742279fa8958a12e519ed)](https://www.codacy.com/app/nukeop/nuclear?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=nukeop/nuclear&amp;utm_campaign=Badge_Grade) [![nuclear](https://snapcraft.io//nuclear/badge.svg)](https://snapcraft.io/nuclear) ![Travis](https://api.travis-ci.org/nukeop/nuclear.svg?branch=master)
+
+<div dir="rtl">
+תוכנה להזרמת שירים מבוססת על מקורות חינמיים
+
+![Showcase](https://i.imgur.com/G9BqIHl.png)
+
+# קישורים
+
+[האתר הרשמי](https://nuclear.js.org)
+
+[Mastodon](https://mstdn.io/@nuclear)
+
+[טוויטר](https://twitter.com/nuclear_player)
+
+ערוץ תמיכה (Matrix): `#nuclear:matrix.org`
+
+ערוץ הדיסקורד: https://discord.gg/JqPjKxE
+
+תרגומים נוספים לדף:
+
+<kbd>[<img title="Português" alt="Português" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/br.svg" width="22">](README-ptbr.md)</kbd>
+<kbd>[<img title="Svenska" alt="Svenska" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/se.svg" width="22">](README-se.md)</kbd>
+<kbd>[<img title="English" alt="English" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/us.svg" width="22">](README.md)</kbd>
+<kbd>[<img title="Hebrew" alt="Hebrew" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/il.svg" width="22">](README-he.md)</kbd>
+<kbd>[<img title="Italiano" alt="Italiano" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/it.svg" width="22">](README-it.md)</kbd>
+
+## מה זה בכלל Nuclear?
+Nuclear זוהי תוכנה חינמית להזרמת תכנים ממקורות חינמיים ברחבי האינטרנט.
+
+מבינכם שמכירים את [mps-youtube](https://github.com/mps-youtube/mps-youtube), זו תוכנה דומה אך עם תוספת של ממשק משתמש גרפי. אנחנו מתמקדים בעיקר על השמע. דמיינו על ספוטיפיי אבל אתם לא צריכים לשלם עבורו ועם ספרייה גדולה יותר.
```

Comment:
A couple of suggestions about מילות היחס:
`אנחנו מתמקדים בעיקר על השמע` -> `אנחנו מתמקדים בעיקר באודיו`
`דמיינו על ספוטיפיי` -> `דמיינו את ספוטיפיי`

And one about making the wording a bit more colloquial:
`אבל אתם לא צריכים לשלם עבורו` -> `אבל ללא תשלום`